esmvaltool/diag_scripts/ocean/diagnostic_tools.py

```diff
@@ -44,7 +57,7 @@ def folder(name):
 
 def get_input_files(cfg, index=0):
     """
-    Load input configuration file.
+    Load input configuration file as a Dictionairy.
```

Comment:
Hi @bouweandela,

I hesitate to make this change right now, as it may wind up being non-trivial to implement and test.  The current method is called by every diagnostic in the ocean diagnostics, so there would be many recipes to run to test it. 

In addition, I'm also working remotely right now, so this kind of change would be hard to implement today. Can we please shift it on a to-do list for a future PR? I'm happy to create a new issue so that I keep it on my to do list.

Reply:
Sure